app/controllers/Api.scala

```diff
@@ -57,8 +57,12 @@ object Api extends Controller with PanDomainAuthActions with SharedSecretAuth {
     val qs: Map[String, Seq[String]] = queryString(req)
 
     CommonAPI.getStubs(qs).asFuture.map {
-      case Left(_) => InternalServerError
-      case Right(contentResponse) => Ok(contentResponse.asJson.noSpaces)
+      case Left(err) =>
```

Comment:
Without opening an IDE I can't tell if this is an exception. If so then it should be in the second param in the logger line. I'm guessing it's a string though.

Reply:
It's an [ApiError](https://github.com/guardian/workflow-frontend/blob/master/app/lib/Response.scala#L8) wrapper object around the HTTP response from the Workflow datastore app